pgex/examples/glow_effect/darkener.py

```diff
@@ -0,0 +1,65 @@
+"""
+This file is a part of the 'Pygame Examples (pgex)' source code.
+The source code is distributed under the MIT license.
+"""
+
+import pygame
+from ._types import Point
+
+
+class Darkener:
+    def __init__(
+        self,
+        base_surf: pygame.Surface,
+        radius: int,
+        darken_ratio: float,
+    ) -> None:
+
+        if darken_ratio > 1 or darken_ratio < 0:
```

Comment:
maybe could use `if not 0 <= darken_ratio <= 1:`, as that seems to add some clarity but this is as you view it, not a necessary change

Reply:
Debatable, the `not` at the beginning can be seen as increasing the complexity.
But in other hand, `a < x < b` is better than `x > a and x < b`